addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -147,6 +147,17 @@ const uniqueRequestId = () => Math.floor(Math.random() * 1000 * 1000 * 1000);
  */
 const buildEmptyResponse = () => new Response(JSON.stringify({ jsonrpc: "2.0", id: uniqueRequestId(), result: {} }));
 
+/**
+ * Fetch the request while adding the X-Do-Not-Track header.
+ * @param request
+ * @returns {Promise<Response>}
+ */
+const fetchWithDoNotTrackHeader = async request => {
+    const headers = new Headers();
+    headers.append("X-Do-Not-Track", "1");
```

Comment:
`Headers` objects are absolutely not required, the `headers` key can just be an object literal (and so this function seems pretty useless).

Reply:
Yes thanks for the tip, I have inlined the function in the code (as there is only one use now) and reduced it to one line using the object literal as parameter as suggested.